### PR TITLE
Fixes for #247

### PR DESCRIPTION
**Pull request by hamishwillee**
This updates the Installing Files document to (firstly) address the issue raised in #247. The problem in #247 was that this doc references droneapi, which has been superseded. On top of that, this document talked a fair bit about virtualenv, but not the solo script commands, which we now hope people will use.

The change adds a section on how to use solo CLI (and recommending it). The virtualenv section has been simplified, positioned as a "manual option" and droneapi has been replaced with dronekit.

### CODE REVIEW

**Comment by mrpollo** [comment]
updated master so it runs tests

File: book/starting-installing.md [hunk 1]
```diff
@@ -86,15 +86,54 @@ We also recommend that you install _git_, as this will be needed to get many of
...
+### Installing packages using the Solo client
 
-This section shows how to install and run packages in an isolated Python environment using  [virtualenv](https://virtualenv.pypa.io/en/latest/).
+The recommended way of working with Python and DroneKit-Python is to use the
+[Solo CLI](starting-utils.html#deployingrunning-dronekit-scripts-on-solo) 
+to package and run the script in a virtual environment.
 
 <aside class="note">
-Solo uses many globally installed packages that may be out of date (in particular, [DroneKit](example-dronekit.html)). Using _virtualenv_ means that you don't need to update the global versions; potentially disturbing native Solo code.
+Solo uses many globally installed packages that may be out of date (in particular, [DroneKit](example-dronekit.html)). Using an isolated environment means that you don't need to update the global versions; potentially disturbing native Solo code.
```
Comment (peterbarker) [inline book/starting-installing.md]: "Using an isolated environment means that you don't need to update the global versions; potentially disturbing native Solo code." semi-colon should be a comma.

Perhaps this:

Using an isolated environment means that the native Solo code will not be disturbed by updates to the global versions.

File: book/starting-installing.md [hunk 2]
```diff
@@ -86,15 +86,54 @@ We also recommend that you install _git_, as this will be needed to get many of
...
 </aside>
 
-First install _virtualenv_ on Solo using _pip_:
+The CLI takes care of packaging all the scripts in a folder:
```
Comment (peterbarker) [inline book/starting-installing.md]: suggest:
 +The CLI takes care of packaging all the scripts in your current folder:

File: book/starting-installing.md [hunk 3]
```diff
@@ -114,16 +153,14 @@ source ./env/bin/activate
 
 You will notice your shell prompt changes to read `(env)root@3dr_solo:`, indicating that you are working in a _virtualenv_.
 
-Now all commands you run from your shell, including launching scripts and installing packages, will only effect this local environment. For example, you can now install a different version of *droneapi* without impacting the global version:
+Now all commands you run from your shell, including launching scripts and installing packages, will only effect this local environment. For example, you can now install a different version of *dronekit* without impacting the global version:
```
Comment (peterbarker) [inline book/starting-installing.md]: That's "affect" *not* "effect"! </grammarnazi>

File: book/starting-installing.md (hunk 1 above)
Comment (hamishwillee) [inline book/starting-installing.md]: I changed this to 

> Solo uses many globally installed packages that may be out of date (in particular, DroneKit), but which cannot be updated without potentially affecting Solo's stability. Using a virtual environment allows you to safely use any package you like.

Comment (hamishwillee) [inline book/starting-installing.md]: and in the pip section I made a change to make the caution more explicit.

> By default, using pip installs or updates packages in the global Python environment. We recommend that instead you install packages into an isolated environment.

**Comment by hamishwillee** [comment]
@peterbarker This should be ready to merge.

**Pull request merged**